macros/WebExtCompat.ejs

```diff
@@ -1,34 +0,0 @@
-<%
```

Comment:
[The macro dashboard](https://developer.mozilla.org/en-US/dashboards/macros) claims this one is used on 0 English pages and 93 non-english pages (although only 29 are on the [results page](https://developer.mozilla.org/en-US/search?locale=*&kumascript_macros=WebExtCompat&topic=none)). Are we OK breaking those pages until translators get a chance to update them?